ants.gemspec

```diff
@@ -22,13 +22,15 @@ Collection of concerns and helpers for Rails + Mongoid + Character web developme
 
   s.add_dependency('mongoid',      '>= 4.0'  )
   s.add_dependency('mongoid-slug', '>= 4.0.0')
+  s.add_dependency('mongoid-history')
+  s.add_dependency('mongoid_search')
   s.add_dependency('devise')
 
   s.add_development_dependency('rails',       [ '>= 4.2'   ])
-  s.add_development_dependency('rspec',       [ '>= 2.0'   ])
-  s.add_development_dependency('rspec-aspic', [ '>= 0.0.2' ])
-  s.add_development_dependency('coveralls')
-
   s.add_development_dependency('bundler', '~> 1.9')
   s.add_development_dependency('rake',    '~> 10.0')
+
+  s.add_development_dependency('capybara-webkit')
```

Comment:
@nkumeiko, please, remove `capybara`, and `capybara-webkit`, we do not have UI tests for Ants, only unit tests.